main.py

```diff
@@ -1,10 +1,56 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_card_file:
+        data_players = json.load(players_card_file)
+        for data_player in data_players.items():
+            player = data_player[1]
+
+            # race
+            race_name = player["race"]["name"]
+            if Race.objects.filter(name=race_name).exists():
+                race = Race.objects.get(name=race_name)
+            else:
+                race = Race.objects.create(
+                    name=race_name,
+                    description=player["race"]["description"]
+                )
+
+            # skills
+            for skills in player["race"]["skills"]:
+                if not Skill.objects.filter(name=skills["name"]).exists():
+                    Skill.objects.create(
+                        name=skills["name"],
+                        bonus=skills["bonus"],
+                        race=race
+                    )
+
+            # guild
+            if player["guild"]:
```

Comment:
you can write like here
![image](https://user-images.githubusercontent.com/104974881/229811462-0c75e6ac-1bf4-416f-9066-1bd35535661a.png)